packages/@expo/metro-config/package.json

```diff
@@ -31,6 +31,7 @@
   ],
   "dependencies": {
     "@expo/config": "~8.0.0",
+    "@expo/styling": "0.0.0",
```

Comment:
based on the flag name, I prefer the name `@expo/css-interop` since it's more clear what it does. Not all "styling" related code lives in "expo/styling".